networkx/readwrite/mermaid.py

```diff
@@ -0,0 +1,238 @@
+"""
+****************
+Mermaid diagrams
+****************
+Read and write NetworkX graphs in Mermaid format.
+
+Mermaid is a collection of text-based diagrams that are well suited
+to be integrated with markdown and get visually rendered by JavaScript.
+
+While Mermaid has plenty of diagram formats [1], here in NetworkX
+we focus on flowcharts [2]
```

Comment:
Can we make it more clear that we only support the flowcharts format? If we add other formats we can change this statement. But the word "focus" confused me into thinking there might be a way to other formats.

Reply:
Fixed, thanks, I was not sure how to phrase that 👍🏾